apps/condo/domains/marketplace/schema/InvoiceContext.js

```diff
@@ -80,25 +92,62 @@ const InvoiceContext = new GQLListSchema('InvoiceContext', {
             schemaDoc: 'How much money do we charge from companies. The percent.',
             isRequired: true,
             type: 'Decimal',
+            access: {
+                read: true,
+                create: userIsAdminOrIsSupport,
+                update: userIsAdminOrIsSupport,
+            },
+            hooks: {
+                resolveInput: ({ resolvedData, fieldPath }) => {
+                    return get(resolvedData, fieldPath, DEFAULT_IMPLICIT_FEE_PERCENT)
+                },
+            },
+        },
+
+        taxRegime: {
+            schemaDoc: 'The regime of counting taxes for company',
+            isRequired: true,
+            type: 'Select',
+            dataType: 'string',
+            options: TAX_REGIMES,
         },
 
-        vat: {
+        vatPercent: {
             schemaDoc: 'The percentage of VAT',
             type: Select,
             dataType: 'integer',
             options: VAT_OPTIONS.map((v) => ({ label: v, value: v })),
             isRequired: false,
         },
 
-        taxRegime: {
-            schemaDoc: 'The regime of counting taxes for company',
+        salesTaxPercent: {
+            schemaDoc: 'The percentage of sales tax',
+            type: 'Decimal',
```

Comment:
You need to validate it better. Should be positive and not more than 100%

Reply:
Done